lib/mix/tasks/ecto.dump.ex

```diff
@@ -17,7 +17,8 @@ defmodule Mix.Tasks.Ecto.Dump do
     quiet: :boolean,
     repo: [:string, :keep],
     no_compile: :boolean,
-    no_deps_check: :boolean
+    no_deps_check: :boolean,
+    dump_prefixes: :string,
```

Comment:
One option here is to use `prefix: [:string, :keep]`. Then you can do `--prefix foo --prefix bar --prefix baz`. Which is how `--repo` works too. It feels this would be a bit more consistent. Thoughts?

Reply:
yeah I like this approach more so that we don't have to enforce strict format for the `dump_prefixes` string. I'll change it to use that 💯